src/main/java/com/redhat/cloud/notifications/routers/NotificationService.java

```diff
@@ -148,4 +149,10 @@ private String getAddress(Principal principal) {
         return resources.deleteEndpointFromDefaults(principal.getAccount(), endpointId)
                 .onItem().transform(ignored -> Response.ok().build());
     }
+
+    @GET
+    @Path("/facets/applications")
+    public Multi<ApplicationFacet> getApplicationsFacets(@Context SecurityContext sec) {
+        return apps.getApplications().onItem().transform(a -> new ApplicationFacet(a.getName(), a.getId().toString()));
```

Comment:
I think this needs for now the `.collectItems().asList().await().indefinitely();` magic like in `EndpointService.getEndpoints)`

Reply:
It seems I can't stop the notifications, but .. don't use ``.await().indefinititely()`` as that's blocking behavior. The use of ``.collectItems().asList()`` is enough and then returning ``Uni<List<ApplicationFacet>>``

.await() should never be used outside tests.

Reply:
Well, apparently the `[][valid data][]` issue we had can be solved by this until a better solution is available.

Reply:
@burmanm hey stranger :-) Yes, .await.indefinitely() is the workaround to the "double" result issue we had when you left :) 
There is a proper fix now https://github.com/quarkusio/quarkus/issues/13317 , but we need to upgrade Quarkus and rollback the workaround.
PR are welcome ! :-P

Reply:
PR is easy, but 1.10.1 universe-bom isn't available in maven-repo yet (even when all the other sub modules are).

https://gist.github.com/burmanm/ce371a45b78b290e77763dd5c0dfbd62